src/cogs/jaegeraccounts.py

```diff
@@ -90,7 +90,8 @@ async def _get_accounts(self):
                             from_time_str = match.group(2)
                             to_time_str = match.group(3)
                         else:
-                            raise InvalidSheetsValue("There seems to be a time (username) formatting error in the google sheets document.")
+                            raise InvalidSheetsValue(
+                                "There seems to be a time (username) formatting error in the google sheets document.")
```

Comment:
Looks like autoformatting, either on my old code or on your end.
```suggestion
                            raise InvalidSheetsValue( "There seems to be a time (username) formatting error in the google sheets document.")
```